ls1intum/Artemis
# `Programming exercises`: Fix issues in the practice mode

### Checklist
#### General


#### Server
#### Client

### Motivation and Context


Fix and improve some minor things about the practice mode.

### Description

This PR fixes a few small things:
1. When the student starts the practice mode, the "Start Practice" button gets disabled and a loading animation appears
2. If the student did not participate before the deadline, the first practice result is already displayed in the result timeline
3. If the student did not participate before the deadline, the clone button no longer shows a not working toggle for the practice mode
4. The student can actually import the practice repository into IntelliJ (before that, IntelliJ assumed the username to be "ab12cde-practice" and did not allow changing it)

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student (your personal TUM-account that you also use as git user on your machine)
- 1 Programming Exercise with a deadline

1. Log in to Artemis
2. Do not participate before the deadline
3. Start the practice mode after the deadline (Check 1, 2 and three)
4. Try to clone the practice repository using IntelliJ/SourceTree

### Review Progress





#### Code Review
#### Manual Tests
